packages/webcert/src/feature/certificate/Buttons/ReplaceCertificateButton.tsx

```diff
@@ -33,13 +36,32 @@ const ReplaceCertificateButton: React.FC<Props> = ({ name, description, enabled,
       confirmButtonDisabled={functionDisabled}>
       <>
         <InfoBox type="observe" activateIconWrap>
-          <p>Om intyget innehåller ett allvarligt fel, till exempel om det är utfärdat på fel patient, bör du istället makulera intyget.</p>
+          <p>
+            {!isDB
+              ? 'Om intyget innehåller ett allvarligt fel, till exempel om det är utfärdat på fel patient, bör du istället makulera intyget.'
+              : 'Om dödsbeviset är utfärdat på fel patient ska du istället makulera dödsbeviset.'}
+          </p>
         </InfoBox>
-        <p>
-          Ett intyg kan ersättas om det innehåller felaktiga uppgifter eller om ny information tillkommit efter att intyget utfärdades. När
-          ett intyg ersätts med ett nytt skapas ett utkast, med samma information som i det ursprungliga intyget, som du kan redigera innan
-          du signerar intyget.{' '}
-        </p>
+        {!isDB ? (
+          <p>
+            Ett intyg kan ersättas om det innehåller felaktiga uppgifter eller om ny information tillkommit efter att intyget utfärdades.
+            När ett intyg ersätts med ett nytt skapas ett utkast, med samma information som i det ursprungliga intyget, som du kan redigera
+            innan du signerar intyget.
+          </p>
+        ) : (
+          <>
+            <p>
+              Ett intyg kan ersättas om det innehåller felaktiga uppgifter eller om ny information tillkommit efter att intyget utfärdades.
+              När ett intyg ersätts med ett nytt skapas ett utkast, med samma information som i det ursprungliga intyget, som du kan
+              redigera innan du signerar intyget.
```

Comment:
Looks like these two paragraphs are identical. Perhaps we could wrap only the next paragraph section in the if-statement.